extensions/qute/runtime/src/main/java/io/quarkus/qute/runtime/EngineProducer.java

```diff
@@ -0,0 +1,169 @@
+package io.quarkus.qute.runtime;
+
+import java.io.InputStream;
+import java.io.InputStreamReader;
+import java.io.Reader;
+import java.nio.charset.Charset;
+import java.util.List;
+import java.util.Optional;
+
+import javax.enterprise.context.ApplicationScoped;
+import javax.enterprise.event.Event;
+import javax.enterprise.inject.Produces;
+import javax.inject.Inject;
+import javax.inject.Singleton;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
```

Comment:
I still see some SL4J logging. Shouldn't it all use JBoss Logging?

Reply:
I agree. I'll search the code again. And again, it's not very useful that SLF4J is always on the class path :-(.